tests/test_selectors.py

```diff
@@ -106,3 +106,30 @@ def test_connection_error(self):
             get_bug.side_effect = requests.exceptions.ConnectionError
             with self.assertWarns(RuntimeWarning):
                 selectors.bug_is_fixed(None, ver)
+
+
+class SkipIfConditionTestCase(unittest.TestCase):
+    """Test :meth: `pulp_smash.selectors.skip_if`."""
+
+    class _DummyClass:  # pylint:disable=missing-docstring,too-few-public-methods
+
+        def __init__(self):
+            self.element = 'test'
+
+        @selectors.skip_if(bool, 'element', True, unittest.SkipTest)
+        def dummy_should_skip(self):  # pylint:disable=missing-docstring,no-self-use
```

Comment:
`pylint:disable` pragmas should be used with care, because - by definition - they ignore warning signs!